glotpress.php

```diff
@@ -35,6 +35,35 @@
 define( 'GP_PLUGIN_FILE', __FILE__ );
 define( 'GP_PATH', dirname( __FILE__ ) . '/' );
 define( 'GP_INC', 'gp-includes/' );
+define( 'GP_WP_REQUIRED_VERSION', '4.4' );
+
+// This adds a row after GlotPress in the plugin page IF an incompatible version of WordPress is running.
+function gp_admin_notices() {
+	global $wp_version;
+	
+	$screen = get_current_screen();
+	
+	if( 'plugins' == $screen->id ) {
+?>
+    <div class="error">
+		<p><?php _e( 'GlotPress Disabled &#151; You are running an unsupported version of WordPress.', 'glotpress' ); ?></p>
+
+		<p><?php echo sprintf( __( 'GlotPress requires WordPress %s or later and has detected you are running %s. Upgrade your WordPress install or deactivate the GlotPress plugin to remove this message.', 'glotpress' ), GP_WP_REQUIRED_VERSION, $wp_version ); ?></p>
+	</div>
+	
+<?php
+	}
+}
+
+global $wp_version;
+
+// Check the WP version, if we don't meet the minimum version to run GlotPress return so we don't cause any errors and add an error to the top of the plugin list.
+if ( !version_compare( $wp_version, GP_WP_REQUIRED_VERSION, '>=' ) ) { 
+	add_action('admin_notices', 'gp_admin_notices', 10, 2);
```

Comment:
* A space after/before the parenthesis is missing.
* `gp_admin_notices` should be more descriptive. What about `gp_unsupported_version_notice`?